int/config/network.go

```diff
@@ -16,18 +16,16 @@ const (
 )
 
 type NetworkInfos struct {
-	Network    string
-	NodeURL    string
-	DNSAddress string
-	Version    string
-	ChainID    uint64
+	Network string
+	NodeURL string
+	Version string
+	ChainID uint64
 }
 
 // NetworkConfig represents the configuration of a network.
 //
 //nolint:tagliatelle
 type NetworkConfig struct {
-	DNS     string   `yaml:"DNS"`
 	URLs    []string `yaml:"URLs"`
 	Default bool     `yaml:"Default"`
 	ChainID uint64   `yaml:"ChainID"`
```

Comment:
Well, in `NetworkConfig`, we have `URLs` that is an array of strings, but it looks like we are always using the first url of the array. So I guess we can use a simple string.
It is not modifiable by users, so there are no issue removing the possibility to have multiple URLs.

IMO, we shouldn't even name those things "Networks" but simply "Nodes". Because, ATM, users can't add custom nodes to the list (as we embed the list inside MS), but as a user, I'd like to be able to add my own node, maybe a local one, or one that is on a remote server, but it will still be connected to Mainnet (or buildnet or testnet....)
So I can have 2 instances of "Mainnet", but one is my node, and one is a node from Massa. And so if for example, my node is down for a maintainance, but I need to connect to MS, I can use Massa's one. 

Definetly not something to do in this PR, but maybe we should open an issue on the subject.

Reply:
https://github.com/massalabs/station/issues/1317